airflow/example_dags/example_bash_operator.py

```diff
@@ -71,5 +71,14 @@
 # [END howto_operator_bash_template]
 also_run_this >> run_this_last
 
+# [START howto_operator_bash_skip]
+this_will_skip = BashOperator(
+    task_id='this_will_skip',
+    bash_command='echo "hello world"; exit 255;',
```

Comment:
I'm not sure we should use 255 for this -- exit codes are not-fully defined, but >128 is usually "I died from a signal".

> POSIX-compatible systems typically use a convention of zero for success and nonzero for error.[10] Some conventions have developed as to the relative meanings of various error codes; for example GNU recommend that codes with the high bit set be reserved for serious errors,[3]. 

So perhaps using 127 would be better. (Exist code is an 8 bit integer)